src/memory.js

```diff
@@ -0,0 +1,42 @@
+
+import loadROM from './utils/loadROM.js';
+
+class Memory {
+
+  constructor(rom) {
+
+    if (!rom) {
+      throw new Error('requires ROM');
+    }
+
+    this.data = loadROM(rom);
+  }
+
+  readROM(value) {
```

Comment:
This is reading a ROM at an address. Could you add some documentation? Do you think `address` or even `addr` or something along those lines would better describe the parameter `value`?

Reply:
sure. I wasn't sure what it was when I started. but can now confidently say I do! :p